### ISSUE TITLE
After i install the logstash the agent become "never connected". but before i install the logstash the agent is active but there was no alerts or information  in the wazuh. 

### ISSUE THREAD

**Issue by sasam3l4** [wazuh/wazuh-documentation#1327]
root@cyber-VirtualBox:/var/ossec/bin# /var/ossec/bin/agent_control -l

Wazuh agent_control. List of available agents:
   ID: 000, Name: cyber-VirtualBox (server), IP: 127.0.0.1, Active/Local
   ID: 001, Name: Windows_7, IP: 192.168.56.104, Pending

List of agentless devices:

root@cyber-VirtualBox:/var/ossec/bin# cat /var/ossec/logs/ossec.log | grep -i -E '(error|warn)'
2019/07/08 10:09:29 ossec-analysisd: ERROR: Read error on /queue/diff/cyber-VirtualBox/533/last-entry: file empty.
2019/07/08 10:21:29 ossec-analysisd: ERROR: Read error on /queue/diff/cyber-VirtualBox/533/last-entry: file empty.
2019/07/08 10:33:30 ossec-analysisd: ERROR: Read error on /queue/diff/cyber-VirtualBox/533/last-entry: file empty.
2019/07/08 10:45:31 ossec-analysisd: ERROR: Read error on /queue/diff/cyber-VirtualBox/533/last-entry: file empty.
2019/07/08 10:57:32 ossec-analysisd: ERROR: Read error on /queue/diff/cyber-VirtualBox/533/last-entry: file empty.
2019/07/08 11:07:23 ossec-analysisd: ERROR: FIM decoder: Bad response 'err Cannot save Syscheck'.
2019/07/08 11:07:23 ossec-analysisd: ERROR: FIM decoder: Bad response 'err Cannot save Syscheck'.
2019/07/08 11:07:24 wazuh-db: ERROR: at wdb_package_insert(): sqlite3_step(): database or disk is full
2019/07/08 11:07:24 wazuh-db: ERROR: Unable to update 'sys_netiface' table for agent '001'
2019/07/08 11:07:24 ossec-analysisd: ERROR: at sc_send_db(): received: 'err Cannot save Network information.'
2019/07/08 11:07:24 ossec-analysisd: ERROR: Unable to send netinfo message to Wazuh DB.
2019/07/08 11:07:24 wazuh-db: ERROR: at wdb_package_insert(): sqlite3_step(): database or disk is full
2019/07/08 11:07:24 wazuh-db: ERROR: Unable to update 'sys_netiface' table for agent '001'
2019/07/08 11:07:24 ossec-analysisd: ERROR: at sc_send_db(): received: 'err Cannot save Network information.'
2019/07/08 11:07:24 ossec-analysisd: ERROR: Unable to send netinfo message to Wazuh DB.
2019/07/08 11:07:24 wazuh-db: ERROR: at wdb_package_insert(): sqlite3_step(): database or disk is full
2019/07/08 11:07:24 wazuh-db: ERROR: Unable to update 'sys_netiface' table for agent '001'
2019/07/08 11:07:24 ossec-analysisd: ERROR: at sc_send_db(): received: 'err Cannot save Network information.'
2019/07/08 11:07:24 ossec-analysisd: ERROR: Unable to send netinfo message to Wazuh DB.
2019/07/08 11:07:24 wazuh-db: ERROR: at wdb_package_insert(): sqlite3_step(): database or disk is full
2019/07/08 11:07:24 wazuh-db: ERROR: Unable to update 'sys_netiface' table for agent '001'
2019/07/08 11:07:24 ossec-analysisd: ERROR: at sc_send_db(): received: 'err Cannot save Network information.'
2019/07/08 11:07:24 ossec-analysisd: ERROR: Unable to send netinfo message to Wazuh DB.
2019/07/08 11:07:24 wazuh-db: ERROR: Deleting old information from 'sys_netiface' table: database or disk is full
2019/07/08 11:07:24 wazuh-db: ERROR: Unable to update 'sys_netiface' table for agent '001'
2019/07/08 11:07:24 ossec-analysisd: ERROR: at sc_send_db(): received: 'err Cannot delete old network information.'
2019/07/08 11:07:24 ossec-analysisd: ERROR: Unable to send netinfo message to Wazuh DB.
2019/07/08 11:07:24 wazuh-db: ERROR: Deleting old information from 'sys_osinfo' table: database or disk is full
2019/07/08 11:07:24 wazuh-db: ERROR: Unable to update 'sys_osinfo' table for agent '001'
2019/07/08 11:07:24 ossec-analysisd: ERROR: at sc_send_db(): received: 'err Cannot save OS information.'
2019/07/08 11:07:25 wazuh-db: ERROR: Deleting old information from 'sys_hwinfo' table: database or disk is full
2019/07/08 11:07:25 wazuh-db: ERROR: Unable to update 'sys_hwinfo' table for agent '001'
2019/07/08 11:07:25 ossec-analysisd: ERROR: at sc_send_db(): received: 'err Cannot save HW information.'
2019/07/08 11:07:25 wazuh-db: ERROR: at wdb_package_insert(): sqlite3_step(): database or disk is full
root@cyber-VirtualBox:/var/ossec/bin#

**Comment by adri9valle** [comment]
Hi @sasam3l4 

In order to give you better assistance, please, could you say me the Wazuh version that you're using? Also, Could you send me the agent logs too, please? 

After all error messages always appear this message:

> 2019/07/08 11:07:25 wazuh-db: ERROR: at wdb_package_insert(): sqlite3_step(): database or disk is full

It seems that the free disk space is about to run out. Please ensure that you have enough free disk space.

Related to the title `but before i install the logstash the agent is active but there was no alerts or information in the wazuh`

This is normal because logstash is the is in charge to forward the alerts to elasticsearch, but the installation of logstash does not affect to the agent's connectivity with the manager, then this is not the problem.

Maybe in the agent logs, we can find the reason why it's happening. 

I'm waiting for the required information in order to help you.

Regards,

Adri,

**Comment by sasam3l4** [comment]
hi @adri9valle 
im sorry but i already uninstall the logstash. and the agent became active again. ill try to install logstash again and ill inform you the result. 

thank you very much.

**Comment by sasam3l4** [comment]
when i tried to install the logstash again this error came up.

root@cyber-VirtualBox:~# sudo apt-get install logstash
E: Could not get lock /var/lib/dpkg/lock-frontend - open (11: Resource temporarily unavailable)
E: Unable to acquire the dpkg frontend lock (/var/lib/dpkg/lock-frontend), is another process using it?
root@cyber-VirtualBox:~# ps aux | grep -i apt
root     22198  0.0  0.0   4628   648 ?        Ss   10:23   0:00 /bin/sh /usr/lib/apt/apt.systemd.daily update
root     22202  0.0  0.0   4628  1548 ?        S    10:23   0:00 /bin/sh /usr/lib/apt/apt.systemd.daily lock_is_held update
root     24729  0.0  0.0  22560  1024 pts/0    S+   10:28   0:00 grep --color=auto -i apt
root@cyber-VirtualBox:~# sudo killall apt apt-get
apt: no process found
apt-get: no process found
root@cyber-VirtualBox:~# sudo apt-get install logstash
E: Could not get lock /var/lib/dpkg/lock-frontend - open (11: Resource temporarily unavailable)
E: Unable to acquire the dpkg frontend lock (/var/lib/dpkg/lock-frontend), is another process using it?
root@cyber-VirtualBox:~#

**Comment by sasam3l4** [comment]
im using this java version

root@cyber-VirtualBox:~# java -version
openjdk version "1.8.0_212"
OpenJDK Runtime Environment (build 1.8.0_212-8u212-b03-0ubuntu1.18.04.1-b03)
OpenJDK 64-Bit Server VM (build 25.212-b03, mixed mode)
root@cyber-VirtualBox:~# javac -version
javac 1.8.0_212

**Comment by sasam3l4** [comment]
localhost:9200/_cat/indices



green  open wazuh-monitoring-3.x-2019.07.08 T0TdSd98Qv2W9Jd5F0brFQ 2 0 4 0  66.8kb  66.8kb
green  open .wazuh                          d0mGegs3RNybq4npwcRLYw 1 0 1 0  11.3kb  11.3kb
yellow open .wazuh-version                  2Dw3dKowRbigOdckGCnnTQ 1 1 1 0   5.2kb   5.2kb
green  open wazuh-monitoring-3.x-2019.07.04 69czEhL5TeOsZUOiH_utPQ 2 0 7 0 116.5kb 116.5kb
green  open .kibana_task_manager            vSNt-wm1RdmTvxH4AkVShg 1 0 2 0  29.5kb  29.5kb
green  open .kibana_1                       XJctX4yhTvWfeKfX0LeBuw 1 0 6 0  50.9kb  50.9kb

**Comment by sasam3l4** [comment]
http://localhost:9200/_cat/templates


.triggered_watches   [.triggered_watches*]      2147483647 
.ml-anomalies-       [.ml-anomalies-*]          0          7010199
.kibana_task_manager [.kibana_task_manager]     0          7010199
.monitoring-kibana   [.monitoring-kibana-7-*]   0          7000199
.ml-state            [.ml-state*]               0          7010199
.management-beats    [.management-beats]        0          70000
.ml-meta             [.ml-meta]                 0          7010199
wazuh                [wazuh-alerts-3.x-*]       0          
.monitoring-alerts-7 [.monitoring-alerts-7]     0          7000199
.monitoring-logstash [.monitoring-logstash-7-*] 0          7000199
.monitoring-beats    [.monitoring-beats-7-*]    0          7000199
.watch-history-9     [.watcher-history-9*]      2147483647 
.logstash-management [.logstash]                0          
.ml-notifications    [.ml-notifications]        0          7010199
.monitoring-es       [.monitoring-es-7-*]       0          7000199
.watches             [.watches*]                2147483647 
.ml-config           [.ml-config]               0          7010199
wazuh-agent          [wazuh-monitoring-3.x-*]   0

**Comment by sasam3l4** [comment]
elasticsearch version

root@cyber-VirtualBox:~# curl "http://localhost:9200/?pretty"
{
  "name" : "cyber-VirtualBox",
  "cluster_name" : "elasticsearch",
  "cluster_uuid" : "J0JmFgabQvez9L5Si7zGTg",
  "version" : {
    "number" : "7.1.1",
    "build_flavor" : "default",
    "build_type" : "deb",
    "build_hash" : "7a013de",
    "build_date" : "2019-05-23T14:04:00.380842Z",
    "build_snapshot" : false,
    "lucene_version" : "8.0.0",
    "minimum_wire_compatibility_version" : "6.8.0",
    "minimum_index_compatibility_version" : "6.0.0-beta1"
  },
  "tagline" : "You Know, for Search"
}

**Comment by sasam3l4** [comment]
i tried this and its successful.

root@cyber-VirtualBox:~# sudo dpkg -i logstash-6.5.4.deb
Selecting previously unselected package logstash.
(Reading database ... 218220 files and directories currently installed.)
Preparing to unpack logstash-6.5.4.deb ...
Unpacking logstash (1:6.5.4-1) ...
Setting up logstash (1:6.5.4-1) ...
Installing new version of config file /etc/logstash/jvm.options ...
Installing new version of config file /etc/logstash/log4j2.properties ...

Configuration file '/etc/logstash/logstash.yml'
 ==> Modified (by you or by a script) since installation.
 ==> Package distributor has shipped an updated version.
   What would you like to do about it ?  Your options are:
    Y or I  : install the package maintainer's version
    N or O  : keep your currently-installed version
      D     : show the differences between the versions
      Z     : start a shell to examine the situation
 The default action is to keep your current version.
*** logstash.yml (Y/I/N/O/D/Z) [default=N] ? N
Using provided startup.options file: /etc/logstash/startup.options
Successfully created system startup script for Logstash

**Comment by sasam3l4** [comment]
hi @adri9valle 
but i dont know after this, what step ishould do after this, please help me. thank you

**Comment by sasam3l4** [comment]
root@cyber-VirtualBox:~# /usr/share/logstash/bin/logstash.yml
bash: /usr/share/logstash/bin/logstash.yml: No such file or directory
root@cyber-VirtualBox:~# find / -iname logstash.yml
find: ‘/run/user/1000/gvfs’: Permission denied
/etc/logstash/logstash.yml


i tried to open the logstash.yml but this happend

**Comment by sasam3l4** [comment]
also itried to start th logstash but the /init.d/logstash in missing


root@cyber-VirtualBox:~#  curl -so /etc/logstash/conf.d/01-wazuh.conf https://raw.githubusercontent.com/wazuh/wazuh/3.7/extensions/logstash/01-wazuh-local.conf
root@cyber-VirtualBox:~#  usermod -a -G ossec logstash
root@cyber-VirtualBox:~# systemctl daemon-reload
root@cyber-VirtualBox:~# systemctl enable logstash.service
Created symlink /etc/systemd/system/multi-user.target.wants/logstash.service → /etc/systemd/system/logstash.service.
root@cyber-VirtualBox:~# systemctl start logstash.service
root@cyber-VirtualBox:~# update-rc.d logstash defaults 95 10
update-rc.d: error: unable to read /etc/init.d/logstash

**Comment by sasam3l4** [comment]
i check the kibana wazuh the i found that the agent status became "never connected"
when i go to the setting of wazuh this error pop up.

Settings. 2012 - Could not save data in elasticsearch due to [cluster_block_exception] blocked by: [FORBIDDEN/12/index read-only / allow delete (api)]; (/elastic/api-hostname/1562211744426)

**Comment by sasam3l4** [comment]
hi @adri9valle 

kibana : 7.1.1
wazuh api : 3.9.2
wazuh agent : 3.9.2
elastic search : 7.1.1
logstash : 6.5.4
filebeat : 7.1.1

i dont know the version of the wazuh manager ad wazuh app. can you give me how can i check the version? 

thankyou

**Comment by adri9valle** [comment]
Hi @sasam3l4 

> i dont know the version of the wazuh manager ad wazuh app. can you give me how can i check the version?

##### Check the Wazuh version
To check the version of the manager you can execute this command `cat /etc/ossec-init.conf`:

```
DIRECTORY="/var/ossec"
NAME="Wazuh"
VERSION="v3.9.2"
REVISION="3925"
DATE="Fri Jun  7 14:28:23 UTC 2019"
TYPE="server"
```


##### Check the Wazuh API version

To check the Wazuh API version you can execute a curl command `curl -u ${user}:${pass} http://${ip}:55000`, for example:

```
# curl -u foo:bar http://localhost:55000?pretty
{
   "error": 0,
   "data": {
      "msg": "Welcome to Wazuh HIDS API",
      "api_version": "v3.9.2",
      "hostname": "master",
      "timestamp": "Mon Jul 08 2019 07:39:20 GMT+0000 (UTC)"
   }
}
```

##### Check the APP version

In the Wazuh Kibana App, go to `Settings > About`:

![image](https://user-images.githubusercontent.com/7983984/60865978-8462c300-a227-11e9-8bfd-b5c4f6af86e2.png)

**NOTE**: To check the agent version, you can do it from the APP, the API or executing the command used to check the Manager version.


Related to the logstash problem, why do you want to use logstash?. Since the elasticsearch 7.x is not necessary to use logstas, we forward the alerts to elasticsearch with filebeat directly.

Logstash can be used in order to transform the data, but if you don't have this necessity I advise you to uninstall logstash.

```
# apt-get purge logstash
```

Once you have uninstalled logstash, you will need to configure filebeat properly, I think that maybe your filebeat configuration is the old configuration that works with logstash, for the 7.x version you need to use  this configuration:

###### Filebeat configuration:
```
curl -so /etc/filebeat/filebeat.yml https://raw.githubusercontent.com/wazuh/wazuh/v3.9.2/extensions/filebeat/7.x/filebeat.yml
```

###### Template configuration:
```
https://raw.githubusercontent.com/wazuh/wazuh/3.9/extensions/elasticsearch/7.x/wazuh-template.json
```

After that restart filebeat, also, please check the templates and indices in elasticsearch:

```
# curl http://172.16.3.2:9200/_cat/templates
...
...
wazuh                       [wazuh-alerts-3.x-*]          0          
wazuh-agent                 [wazuh-monitoring-3.x-*]      0          
...
....
```

```
# curl http://172.16.3.2:9200/_cat/indices
yellow open .wazuh                           6Q75f2PqQMi6Tx6ui4o-2Q 1 1    1 0  11.8kb  11.8kb
green  open wazuh-monitoring-3.x-2019.07.02  gZAGEBOaRsuqO6WoL0raHw 2 0    0 0    566b    566b
green  open .kibana_2                        l9H0ZcKsQ92sJHopr-YeoQ 1 0  707 0 322.8kb 322.8kb
green  open wazuh-alerts-3.x-2019.07.05      kZbLP-QlQaGYEcM4HyO_tg 3 0 4728 0 583.4kb 583.4kb
green  open .kibana_task_manager             TJEaYUVRRsOP1m58WKHQ9A 1 0    2 0  46.3kb  46.3kb
yellow open filebeat-7.2.0-2019.07.02-000001 IzvALh8qTJy2OXPlfS57LA 1 1    0 0    283b    283b
green  open wazuh-alerts-3.x-2019.07.08      6zcdljSeSISFIIk4dL2tww 3 0 2438 0 726.2kb 726.2kb
yellow open .wazuh-version                   6bYWF0VuQeaPDiweeuAIxw 1 1    1 0   5.2kb   5.2kb
green  open wazuh-monitoring-3.x-2019.07.08  evfyw91IRBSX1vBAIEwuqw 2 0    3 0  49.1kb  49.1kb
green  open .kibana_1                        u2qg0Lp9TmyCpaSEjJ39Ww 1 0    6 0  63.4kb  63.4kb
green  open wazuh-alerts-3.x-2019.07.02      QD2Fbq_0RH281mVe2LnVuw 3 0  162 0 488.3kb 488.3kb
green  open wazuh-monitoring-3.x-2019.07.05  MXyDFxNHTRisWwRe4_Oddw 2 0    0 0    566b    566b
```

Please try to do this, and let me know if it works for you. 

Regards,

Adri,

**Comment by sasam3l4** [comment]
hi @adri9valle 

the template configuration says that no such file or directory. shall i use the curl command used in filebeat configuration in template configuration?

**Comment by sasam3l4** [comment]
in my  http://localhost:9200/_cat/indices

green  open wazuh-monitoring-3.x-2019.07.08 T0TdSd98Qv2W9Jd5F0brFQ 2 0 4 0  66.8kb  66.8kb
green  open .wazuh                          d0mGegs3RNybq4npwcRLYw 1 0 1 0  11.3kb  11.3kb
yellow open .wazuh-version                  2Dw3dKowRbigOdckGCnnTQ 1 1 1 0   5.2kb   5.2kb
green  open wazuh-monitoring-3.x-2019.07.04 69czEhL5TeOsZUOiH_utPQ 2 0 7 0 116.5kb 116.5kb
green  open .kibana_task_manager            vSNt-wm1RdmTvxH4AkVShg 1 0 2 0  29.5kb  29.5kb
green  open .kibana_1                       XJctX4yhTvWfeKfX0LeBuw 1 0 6 0  50.9kb  50.9kb



in  http://localhost:9200/_cat/templates

.triggered_watches   [.triggered_watches*]      2147483647 
.ml-anomalies-       [.ml-anomalies-*]          0          7010199
.kibana_task_manager [.kibana_task_manager]     0          7010199
.monitoring-kibana   [.monitoring-kibana-7-*]   0          7000199
.ml-state            [.ml-state*]               0          7010199
.management-beats    [.management-beats]        0          70000
.ml-meta             [.ml-meta]                 0          7010199
wazuh                [wazuh-alerts-3.x-*]       0          
.monitoring-alerts-7 [.monitoring-alerts-7]     0          7000199
.monitoring-logstash [.monitoring-logstash-7-*] 0          7000199
.monitoring-beats    [.monitoring-beats-7-*]    0          7000199
.watch-history-9     [.watcher-history-9*]      2147483647 
.logstash-management [.logstash]                0          
.ml-notifications    [.ml-notifications]        0          7010199
.monitoring-es       [.monitoring-es-7-*]       0          7000199
.watches             [.watches*]                2147483647 
.ml-config           [.ml-config]               0          7010199
wazuh-agent          [wazuh-monitoring-3.x-*]   0

**Comment by adri9valle** [comment]
Hi @sasam3l4 

> wazuh [wazuh-alerts-3.x-] 0

It seems that the template is created but the index is not.

Please could you send me your filebeat configuration?

```
cat /etc/filebeat/filebeat.yml
```

Regars,

**Comment by sasam3l4** [comment]
# Wazuh - Filebeat configuration file

filebeat.inputs:
  - type: log
    paths:
      - '/var/ossec/logs/alerts/alerts.json'

setup.template.json.enabled: true
setup.template.json.path: "/etc/filebeat/wazuh-template.json"
setup.template.json.name: "wazuh"
setup.template.overwrite: true

processors:
  - decode_json_fields:
      fields: ['message']
      process_array: true
      max_depth: 200
      target: ''
      overwrite_keys: true
  - drop_fields:
      fields: ['message', 'ecs', 'beat', 'input_type', 'tags', 'count', '@version', 'log', 'offset', 'type', 'host']
  - rename:
      fields:
        - from: "data.aws.sourceIPAddress"
          to: "@src_ip"
      ignore_missing: true
      fail_on_error: false
      when:
        regexp:
          data.aws.sourceIPAddress: \b\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}\b
  - rename:
      fields:
        - from: "data.srcip"
          to: "@src_ip"
      ignore_missing: true
      fail_on_error: false
      when:
        regexp:
          data.srcip: \b\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}\b
  - rename:
      fields:
        - from: "data.win.eventdata.ipAddress"
          to: "@src_ip"
      ignore_missing: true
      fail_on_error: false
      when:
        regexp:
          data.win.eventdata.ipAddress: \b\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}\b

# Send events directly to Elasticsearch
output.elasticsearch:
  hosts: ['http://192.168.56.108:9200']
  #pipeline: geoip
  indices:
    - index: 'wazuh-alerts-3.x-%{+yyyy.MM.dd}'

# Optional. Send events to Logstash instead of Elasticsearch
#output.logstash.hosts: ["YOUR_LOGSTASH_SERVER_IP:5000"]

**Comment by adri9valle** [comment]
Hi @sasam3l4 

The filebeat configuratio seems to be ok. Could you check in the template configuration `/etc/filebeat/wazuh-template.json` the parameter `full_log`? The `full_log` should be set as text type if you follow the documentation maybe is set as object type.

Also, could you check the elasticsearch log? `/var/log/elasticsearch/my-cluster.log` if the events are being received but cannot be indexed for any reason we could see possible errors here.

Regards,

**Comment by sasam3l4** [comment]
"full_log":{
      "type": "text"
}

in /var/log/elasticsearch/
i cant see "my-cluster.log" file.

**Comment by sasam3l4** [comment]
![1](https://user-images.githubusercontent.com/52516886/60931901-6e700380-a2f6-11e9-9f01-14e7e13b6f64.PNG)

**Comment by sasam3l4** [comment]
hi @adri9valle 

I look for the logs of elasticsearch and this is what i found.
![2](https://user-images.githubusercontent.com/52516886/60933485-68c9ec00-a2fd-11e9-989f-467c1604d946.PNG)

**Comment by adri9valle** [comment]
Hi @sasam3l4 

> ... flood stage disk watermark [95%] exceeded on ... 

It seems that the free disk space is at its limit and elasticsearch does not permit to write new data in the indices. 

Please, could you check your free disk space with this command  `df -h`? 

```
df -h
Filesystem      Size  Used Avail Use% Mounted on
/dev/sda1        40G   13G   25G  34% /
devtmpfs        1,9G     0  1,9G   0% /dev
tmpfs           1,9G     0  1,9G   0% /dev/shm
tmpfs           1,9G  8,5M  1,9G   1% /run
tmpfs           1,9G     0  1,9G   0% /sys/fs/cgroup
tmpfs           379M     0  379M   0% /run/user/1000
```

After that please check that you have enough disk space.

Please, let me know if it works for you.

Regards,

**Comment by sasam3l4** [comment]
hi @adri9valle 

here is my disk space information

> root@rootcyber-VirtualBox:~# df -h
Filesystem      Size  Used Avail Use% Mounted on
udev            4.5G     0  4.5G   0% /dev
tmpfs           908M  9.2M  899M   2% /run
/dev/sda1        21G  7.1G   13G  37% /
tmpfs           4.5G   83M  4.4G   2% /dev/shm
tmpfs           5.0M  4.0K  5.0M   1% /run/lock
tmpfs           4.5G     0  4.5G   0% /sys/fs/cgroup
tmpfs           908M   64K  908M   1% /run/user/1000
/dev/sr0         82M   82M     0 100% /media/rootcyber/VBox_GAs_6.0.4
root@rootcyber-VirtualBox:~# free -m
              total        used        free      shared  buff/cache   available
Mem:           9074        3405        1499         125        4169        5212

**Comment by sasam3l4** [comment]
i tried to create a virtual boxwith 8gb of memory and 20gb of storage, then install from the start. but still the same. no index created but template was created.

**Comment by sasam3l4** [comment]
[Untitled Document 1.txt](https://github.com/wazuh/wazuh-documentation/files/3385453/Untitled.Document.1.txt)
### here is the result of /var/log/elasticsearch/elasticsearch.log

**Comment by adri9valle** [comment]
Hi @sasam3l4 

> also i tried to do this lsof /var/ossec/logs/alerts/alerts.json

Could you check the filebeat log:

```
tail -f /var/log/filebeat/filebeat
```

Please also, could you stop filebeat and start it in debug mode?

```
systemctl stop filebeat
```

And:

```
filebeat -e -d "*"
```

The previous command will launch filebeat in foreground and will display all the logs. Please check this logs and paste here the output.

Regards,

Adri,

**Comment by sasam3l4** [comment]
hi @adri9valle 

> Could you check the filebeat log:
tail -f /var/log/filebeat/filebeat

here's the result

> root@rootcyber-VirtualBox:~# tail -f /var/log/filebeat/filebeat
tail: cannot open '/var/log/filebeat/filebeat' for reading: No such file or directory
tail: no files remaining

**Comment by sasam3l4** [comment]
hi @adri9valle 

> Please also, could you stop filebeat and start it in debug mode?

here's the result.
[filebeat log.txt](https://github.com/wazuh/wazuh-documentation/files/3394779/filebeat.log.txt)

**Comment by sasam3l4** [comment]
Hi @adri9valle 

I find and error that cays it cant connect to the ip:9200
thats why i tried to change the filebeat.yml to localhost:9200 and it worked. 
thank you so much for your help.

regards,
samela

**Comment by adri9valle** [comment]
Hi @sasam3l4,

I'm very glad to know that it works for you properly now. If you have more questions don't hesitate to open a new issue, or re-open this.

Regards,

Adri,
